engine/table/src/main/java/io/deephaven/engine/table/impl/DeferredViewTable.java

```diff
@@ -307,6 +306,13 @@ public static abstract class TableReference extends LivenessArtifact implements
             }
         }
 
+        /**
+         * Is the node updating?
+         *
+         * @return true if the node is updating; false otherwise.
+         */
+        public abstract boolean isRefreshing();
```

Comment:
It seems to me that this need not be abstract; we take a `table` in the constructor, don't we? I imagine we could write down at that time if it is refreshing or not.